cmake/std/atdm/README.md

```diff
@@ -492,14 +492,18 @@ options are read.
 
 The specific `cmake/std/atdm/<system-name>/` sub-directories and the systems
 they support are:
+* `chama/`: Supports SNL HPC machine `chama/`.
+
 * `mutrino/`: Supports SNL HPC machine `mutrino/`.
 
 * `rhel6/`: RHEL6 systems with the SEMS NFS environment
 
 * `ride/`: Supports GNU and CUDA builds on both the SRN machine `ride` and the
   mirror SON machine `white`.
 
+* `serrano/`: Supports SNL HPC machine `serrano/`.
```

Comment:
Same here.  Remove `/` from "Supports SNL HPC machine `serrano/`"  And can we remove `/` from "Supports SNL HPC machine `mutrino/`" above?  I missed that on a prior review.